vunit/ui.py

```diff
@@ -426,7 +426,10 @@ def add_source_files_from_csv(self, project_csv_path, vhdl_standard=None):
                 files = libs_and_files[lib_name]
                 lib = self.add_library(lib_name)
                 list_of_source_files.append(lib.add_source_files(files))
-            return list_of_source_files
+            
+            list_of_source_files_flattened = [val for sublist in list_of_source_files for val in sublist]
```

Comment:
+= would have been nicer. Also it  seems this should have a unit test since there were so many bugs.